lib/rendition.js

```diff
@@ -248,6 +251,33 @@ class Rendition {
             console.log("could not determine encoding/charset");
         }
 
+        // return here for pipeline rendition
```

Comment:
please add notes and "TODO's" everywhere you added code only for the sdk so we remember to either:
- use it in the pipeline
- or remove it when we move away from the SDK completely